src/App.js

```diff
@@ -33,7 +44,12 @@ function App() {
             </Route>
           </Switch>
         </main>
-        <footer className="Footer"></footer>
+        <Switch>
+          <Route exact path="/"></Route>
+          <Route path="/*">
```

Comment:
To be more exact, you could give here an array of strings for every matching path:
https://reactrouter.com/web/api/Route/path-string-string